vscode/package.json

```diff
@@ -285,11 +280,11 @@
         "icon": "$(play)"
       },
       {
-        "command": "qsharp-vscode.debugEditorContentsWithCircuit",
-        "title": "Debug Q# File with Circuit Diagram",
+        "command": "qsharp-vscode.runEditorContentsWithCircuit",
+        "title": "Run file and show circuit diagram",
```

Comment:
I think it's still good to have Q# in the command text. I usually include that when searching for commands.

Reply:
Good point... I was copying from the command below it, but those commands end up with a "Q#" prefix in the Command Palette while this one isn't. I'll update.

![image](https://github.com/microsoft/qsharp/assets/16928427/e92d7d7a-3d45-46ce-b3be-ca2b326425aa)